package.json

```diff
@@ -1,23 +1,53 @@
 {
   "name": "msk-insight",
-  "version": "0.1.0",
+  "version": "0.0.0",
   "private": true,
+  "dependencies": {
+    "@types/classnames": "^2.2.9",
+    "@types/jquery": "^3.3.29",
+    "@types/lodash": "^4.14.123",
+    "@types/node": "^11.11.1",
+    "@types/rc-tooltip": "^3.7.1",
+    "@types/react": "^16.8.7",
+    "@types/react-dom": "^16.8.2",
+    "@types/react-router-bootstrap": "^0.24.5",
+    "@types/react-table": "^6.8.1",
+    "@types/seamless-immutable": "^7.1.10",
+    "@types/superagent": "^4.1.1",
+    "autobind-decorator": "^2.4.0",
+    "bootstrap": "^4.3.1",
+    "cbioportal-frontend-commons": "^0.1.0",
+    "classnames": "^2.2.6",
+    "csvtojson": "^2.0.10",
+    "font-awesome": "^4.7.0",
+    "lodash": "^4.17.13",
+    "mobx": "^3.1.7",
+    "mobx-react": "^4.1.3",
+    "rc-tooltip": "^3.7.3",
+    "react": "^16.8.6",
+    "react-bootstrap": "^1.0.0-beta.6",
+    "react-dom": "^16.8.6",
+    "react-mutation-mapper": "0.3.6",
+    "react-router-bootstrap": "^0.24.4",
+    "react-router-dom": "^4.3.1",
+    "react-table": "^6.10.0",
+    "serve": "^11.3.0",
+    "typeface-open-sans": "^0.0.54",
+    "typescript": "^3.3.3333"
+  },
+  "resolutions": {},
   "engines": {
     "node": "8.12.0"
   },
   "scripts": {
-    "build": "yarn run build-client && yarn run build-server && yarn run copy-reactapp",
-    "copy-reactapp": "node copy-reactapp.js",
-    "build-client": "yarn --cwd client install && yarn --cwd client run build",
-    "build-server": "yarn --cwd server install && yarn --cwd server run build",
-    "start": "node server/dist/server.js",
-    "concurrent": "concurrently \"yarn run start-server\" \"yarn run start-client\"",
-    "start-server": "node start-server.js",
-    "start-client": "node start-client.js"
+    "start": "serve -s build",
```

Comment:
Added this to make the build work under `heroku`. But we should switch to netlify soon since this is a completely static website now.